Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/InitialContent.md

```diff
@@ -287,6 +287,20 @@ which renders in:
 
 ![Toolkit logo](https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/Microsoft.Toolkit.Uwp.SampleApp/Assets/ToolkitLogo.png)
 
+You can also specify image width like this:
```

Comment:
We should follow the current spec extension from here (though there doesn't seem to be a standard, apparently some third parties implement `=WxH` already): https://github.com/commonmark/CommonMark/wiki/Deployed-Extensions#image-size

Other references: https://stackoverflow.com/questions/14675913/how-to-change-image-size-markdown, https://talk.commonmark.org/t/support-for-image-dimensions/148/4